timers.js

```diff
@@ -1,4 +1,35 @@
+var seconds = 0;
+var timerId = 0;
+var timerDisplay = document.getElementById("timer");
+var numClicks = 0;
 
+var reset = document.getElementById("reset");
+var resetClickEvent = function(){
+  clearInterval(timerId);
+  seconds = 0;
+  document.getElementById("timer").innerHTML = "Stop Watch";
+  numClicks = 0;
+}
+reset.onclick = resetClickEvent;
 
+var start = document.getElementById("start");
+var startClickEvent = function(){
+  numClicks++
+  if (numClicks == 1) {
+    timerDisplay.innerHTML = (seconds + " seconds");
+    timerId = setInterval(updateTime, 1000);
+  }
+}
+start.onclick = startClickEvent;
 
+var pause = document.getElementById("pause");
+var pauseClickEvent = function(){
+  clearInterval(timerId);
+  numClicks = 0;
+}
+pause.onclick = pauseClickEvent;
```

Comment:
Nothing wrong with using on `.onclick`, but keep in mind that you cannot add multiple click events to an element using this method. `.onclick` is essentially an object property, so it can only have one value.

Personally a much bigger fan of `.addEventListener` (JS) and `.on` or `.click` (jQuery).